docs/content/functioneel/zaaktypen_api.md

```diff
@@ -0,0 +1,188 @@
+ZAAKTYPEN API
+===============================================
+**FUNCTIONELE DOCUMENTATIE**
+
+[aanzet tot documentatie van de API’s van het ZDS2-koppelvlak. Is vooralsnog een
+‘try-out’ n.a.v. [userstory
+\#188](https://github.com/VNG-Realisatie/gemma-zaken/issues/188) om inzicht te
+krijgen in de wijze van (functioneel) documenteren van API’s, als onderdeel van
+koppelvlakdocumentatie]
+
+| **Aspect**      | **Beschrijving**                                                                                                                                                                                  |
+|-----------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Doel            | Het kunnen aanspreken van een voorziening voor het onderhouden en raadplegen van zaaktypecatalogus (ZTC) inclusief daarinopgenomen zaaktypen, statustypen, roltypen en informatieobjecttypen, rollen en relaties naar betrokkenen, objecten, zaaktypen en statustypen. |
+| Domein          | Zaakgericht werken                                                                                                                                                                                |
+| Provider        | [Zaaktypecataloguscomponent](https://www.gemmaonline.nl/index.php/Omgevingswet/1.5/id-3ef9cdd9-631c-4d3e-88c3-f756423d6314) (GEMMA2)                                                              |
```

Comment:
Volgens mij link je naar de verkeerde component (die in het Omgevingewetdomein) en moet het die naar GEMMA zijn. Heb ik aangepast.